AdministratorAdministration+forwardings.yaml

```diff
@@ -0,0 +1,499 @@
+    initiating-requests:
+    consequent-requests:
+      - client-name: ApplicationLayerTopology://v1/delete-ltp-and-dependents
+        uuid: aa-1-0-0-op-c-bm-alt-1-0-0-002
+
+  - forwarding-name: OamRequestCausesFcUpdateRequest
+    uuid: aa-1-0-0-op-fc-bm-013
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: aa-1-0-0-op-s-bm-009
+      fc-port-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: aa-1-0-0-op-s-bm-009
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+    consequent-requests:
+      - client-name: ApplicationLayerTopology://v1/update-fc
+        uuid: aa-1-0-0-op-c-bm-alt-1-0-0-003
+
+  - forwarding-name: OamRequestCausesFcPortUpdateRequest
+    uuid: aa-1-0-0-op-fc-bm-014
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: aa-1-0-0-op-s-bm-009
+      fc-port-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: aa-1-0-0-op-s-bm-009
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+    consequent-requests:
+      - client-name: ApplicationLayerTopology://v1/update-fc-port
+        uuid: aa-1-0-0-op-c-bm-alt-1-0-0-004
+
+  - forwarding-name: OamRequestCausesFcPortDeletionRequest
+    uuid: aa-1-0-0-op-fc-bm-015
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: aa-1-0-0-op-s-bm-009
+      fc-port-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: aa-1-0-0-op-s-bm-009
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+    consequent-requests:
+      - client-name: ApplicationLayerTopology://v1/delete-fc-port
+        uuid: aa-1-0-0-op-c-bm-alt-1-0-0-005
+
+  - forwarding-name: NewApplicationCausesRequestForInquiringOamRequestApprovals
+    uuid: aa-1-0-0-op-fc-is-000
+    forwarding-type: ProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/regard-application
+          uuid: aa-1-0-0-op-s-is-001
+      fc-port-update:
+        - server-name: /v1/regard-application
+          uuid: aa-1-0-0-op-s-is-001
+      fc-port-deletion:
+        - server-name: /v1/disregard-application
+          uuid: aa-1-0-0-op-s-is-002
+      operation-client-deletion:
+        - server-name: /v1/disregard-application
+          uuid: aa-1-0-0-op-s-is-002
+    initiating-requests:
+        - server-name: /v1/regard-application
+          uuid: aa-1-0-0-op-s-is-001
+    consequent-requests:
+      - client-name: RegistryOffice:://v1/inquire-oam-request-approvals
```

Comment:
As per my understanding, /v1/inquire-oam-request-approvals (operation-client under each application) is applicable only for AA application, changing the api-segment of all occurences of /v1/inquire-oam-request-approvals to "individual management (im)" would be more precise.

Kindly let me know your views

Reply:
Changed as proposed.